tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_RRFS_CONUScompact_25km_ics_RRFS_lbcs_RRFS_suite_RRFS_sas.yaml

```
@@ -0,0 +1,29 @@
+metadata:
+  description: |-
+    This test is to ensure that the workflow running in community mode
+    completes successfully on the RRFS_CONUScompact_25km grid using the RRFS_v1beta
+    physics suite. It uses RRFS forecasts mapped onto 3-km regular grid (rrfs*.conus.grib2) for
+    ICs and LBCs. This test uses old v1 sfc_data.nc format, not the v2 sfc_data.
+user:
+  RUN_ENVIR: community
+workflow:
+  CCPP_PHYS_SUITE: RRFS_sas
+  PREDEF_GRID_NAME: RRFS_CONUScompact_25km
+  DATE_FIRST_CYCL: '2024060517'
+  DATE_LAST_CYCL: '2024060517'
+  FCST_LEN_HRS: 3
+  PREEXISTING_DIR_METHOD: rename
+task_get_extrn_ics:
+  EXTRN_MDL_NAME_ICS: RRFS
+  FV3GFS_FILE_FMT_ICS: grib2
+  USE_USER_STAGED_EXTRN_FILES: true
+task_get_extrn_lbcs:
+  EXTRN_MDL_NAME_LBCS: RRFS
+  LBC_SPEC_INTVL_HRS: 1
+  FV3GFS_FILE_FMT_LBCS: grib2
+  USE_USER_STAGED_EXTRN_FILES: true
+task_plot_allvars:
+  COMOUT_REF: ""
+rocoto:
+  tasks:
+    taskgroups: '{{ ["parm/wflow/prep.yaml", "parm/wflow/coldstart.yaml", "parm/wflow/post.yaml", "parm/wflow/plot.yaml"]|include }}'
```

Since this test doesn't test plotting, is there a reason to include `parm/wflow/plot.yaml`?

```suggestion
    taskgroups: '{{ ["parm/wflow/prep.yaml", "parm/wflow/coldstart.yaml", "parm/wflow/post.yaml"]|include }}'
```

---

Updated in the recent commits.